.github/workflows/pull_request.yml

```diff
@@ -8,39 +8,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        name: "Checkout"
       - name: Push Docker Image to Github Registry
         uses: whoan/docker-build-with-cache-action@v5
         with:
           username: "${{ github.actor }}"
           password: "${{ secrets.GITHUB_TOKEN }}"
-          image_name: ${{ secrets.flytegithub_repo }}/flytepropeller
-          image_tag: ${{ github.sha }}
-          push_git_tag: true
+          image_name: lyft/flytepropeller
```

Comment:
If you would like to get rid of this action, I would say let's use the github-provided [cache action](https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows).

so the action would be something like:
```
- uses: github/cache
   key: "latest"
   path: "/docker_cache"
```

and in the script below (end2end or otherwise), we should follow [this](https://runnable.com/blog/distributing-docker-cache-across-hosts):
```
docker load /docker_cache/latest.tar
docker build...
docker save lyft/flytepropeller:<sha> $(docker history -q lyft/flytepropeller) > /docker_cache/latest.tar
```

If you see some other way to leverage caching please elaborate on how will this work?